default/scripting/techs/defense/mines.macros

```diff
@@ -271,3 +190,17 @@ EG_SYSTEM_MINES
                     empire = Target.Owner
             ]
 '''
+
+SYSTEM_CONTAINS_SOURCE_OR_OWNED_PLANET
+'''                ContainedBy And [
+                    System 
+                    Contains Or [
+                        Source
```

Comment:
Why is the Source condition used?

Wouldn't the And [] case cover that (assuming the source is a planet)?